src/org/eclipse/swt/internal/c/XInput.d

```diff
@@ -49,12 +60,6 @@ alias XDeviceButtonEvent XDeviceButtonReleasedEvent;
 alias XDeviceButtonEvent XDeviceButtonPressedEvent;
 alias XDeviceKeyEvent XDeviceKeyReleasedEvent;
 alias XDeviceKeyEvent XDeviceKeyPressedEvent;
-alias void function(void *, char *, int, int, char * *) _BCD_func__1398;
-alias int function(void *) _BCD_func__1440;
-alias int function(void *, XErrorEvent *) _BCD_func__1441;
-alias void function(void *, char *, char *) _BCD_func__1515;
-alias int function(void *, char *, char *) _BCD_func__1516;
-alias void function(void *, char *, char *) _BCD_func__1517;
```

Comment:
I was trying to fix these types, but didn't know the identity of functions. Moreover, They are not used.